redisearch/client.py

```diff
@@ -210,7 +209,7 @@ def create_index(self, fields, no_term_offsets=False,
         - **stopwords**: If not None, we create the index with this custom stopword list. The list can be empty
         """
 
-        args = [self.CREATE_CMD, self.index_name]
+        args = [self.CREATE_CMD, self.index_name, 'ON', 'HASH']
```

Comment:
Shouldn't we add it as an option?
Otherwise, it looks good!

Reply:
I don't see the point to add it now before we have other options, we can always add it as optional argument with defaul (as you did on your PR)